frontend/catalyst/ag_primitives.py

```diff
@@ -522,19 +503,22 @@ def get_source_code_info(tb_frame):
 # issues such as always tracing through code that should only be executed conditionally. We might
 # have to be even more restrictive in the future to prevent issues if necessary.
 module_allowlist = (
-    config.DoNotConvert("pennylane"),
-    config.DoNotConvert("catalyst"),
-    config.DoNotConvert("jax"),
-) + config.CONVERSION_RULES
+    ag_config.DoNotConvert("pennylane"),
+    ag_config.DoNotConvert("catalyst"),
+    ag_config.DoNotConvert("jax"),
+    *ag_config.CONVERSION_RULES,
+)
 
 
 def converted_call(fn, args, kwargs, caller_fn_scope=None, options=None):
     """We want AutoGraph to use our own instance of the AST transformer when recursively
     transforming functions, but otherwise duplicate the same behaviour."""
 
+    # TODO: eliminate the need for patching by improving the autograph interface
```

Comment:
Yes exactly, one on my goals would be to improve the interface in Malt so that downstream packages can hook into the existing classes and configurations while allowing extensions and modifications, otherwise it either requires copying some code from the malt package that you want to reuse or patching like we do here.